packages/formation/sass/modules/_m-promo-banner.scss

```diff
@@ -0,0 +1,57 @@
+.vads-c-promo-banner {
```

Comment:
Is it worth separating the positioning here from the styling? Granted, we only have the one use case for it now, but the style of the banner is independent of where it's positioned on the screen and putting those styles in two separate classes might make it more flexible later.

Like `vads-c-promo-banner` for the visual styles and `vads-c-promo-banner--fixed-bottom` to fix its position to the bottom of the page. 

Curious what @jaredcunha-usds thinks.

Reply:
Oh hm, that makes sense to me. That's a pretty quick change...

Reply:
I think we can leave the default styling be fixed at the bottom, so I don't think we need to create the variant. Maybe a variant for whatever changes in the future, but I don't think we'll need `vads-c-promo-banner--fixed-bottom`

Reply:
Kind of interesting seeing it without the `vads-c-promo-banner--fixed-bottom` applied -

![image](https://user-images.githubusercontent.com/1915775/59800302-3f94dd80-92b4-11e9-9dca-de900faf33f0.png)

Reply:
@jaredcunha-usds whoops! Should I revert the commit?

Reply:
@ncksllvn Yeah, I think so. 😬